### ISSUE TITLE
Implement proper authorization in mutations

### ISSUE THREAD

**Issue by aerugo** [Edgeryders-Participio/realities#18]
Authorization should be included in all mutations to determine if user may preform the action.
Rules are specified in the white paper at bit.ly/realities-whitepaper and in the map at https://bitly.com/realitiesboard

**Comment by hspeijer** [comment]
Check out https://grandstack.io/docs/guide-middleware.html

**Comment by erikfrisk** [comment]
@hspeijer: That link talks about how we were doing auth prior to the [refactor](https://github.com/theborderland/realities/pull/51). We just didn't happen to use the `user` context property in any of the query fields where we used neo4j-graphql-js since our API is readable to the public. 

In the refactor I introduced the [graphql-resolvers](https://github.com/lucasconstantino/graphql-resolvers) library for auth as recommended/suggested by [graphql-tools](https://www.apollographql.com/docs/graphql-tools/). 

You've probably seen all this already, but commenting here so we have everything in one place and in case others get involved down the line :)
